src/platform/ASR/ASROTAImageProcessor.h

```diff
@@ -69,4 +69,7 @@ class ASROTAImageProcessor : public OTAImageProcessorInterface
     const char * mImageFile = nullptr;
 };
 
+#define LEGA_OTA_OK 0            // No error, operation OK
```

Comment:
Do these need to be exposed globally in a header? I would have expected these to be exposed by `lega_ota_*` headers.

Otherwise it seems that placing these in C++ file is better and please make them C++ constants `static constexpr int32_t kLegaOtaOk = 0` and such.

Reply:
No need to put these in a header file, resolved it!